RevenueCatUI/CustomerCenter/Views/CustomerCenterNavigationLink.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  CustomerCenterNavigationLink.swift
+//  RevenueCat
+//
+//  Created by Facundo Menzella on 14/1/25.
+//
+
+import RevenueCat
+import SwiftUI
+
+#if os(iOS)
+
+/// A view that provides a navigation link to `CustomerCenterView` with a customizable label.
+///
+/// This is the **preferred way** to integrate `CustomerCenterView` into your `NavigationView`,
```

Comment:
nitpick: double-backticks so that these automatically turn into links:
```suggestion
/// A view that provides a navigation link to ``CustomerCenterView`` with a customizable label.
///
/// This is the **preferred way** to integrate ``CustomerCenterView`` into your `NavigationView`,
```